pkg/memory-monitor/src/statistics-collector/main.c

```diff
@@ -0,0 +1,108 @@
+#include <fcntl.h>
```

Comment:
Copyright?

Reply:
Oops, will add. I though I had already done it.